common/src/config.rs

```diff
@@ -99,13 +119,13 @@ pub struct EpochPublisherSet {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct RangeServerConfig {
     pub range_maintenance_duration: time::Duration,
-    pub proto_server_port: u32,
-    pub fast_network_port: u32,
+    pub proto_server_addr: HostPort,
+    pub fast_network_addr: HostPort,
```

Comment:
Is this going to work with multiple range servers? Are they supposed to be the local addresses to listen on but not how other components talk to the range server?

Reply:
Currently no, there's only 1 rangeserver entry in the config (not an array).
In my opinion, we should separate the portion of the config defining listen addresses vs defining external addresses to contact other components.

Reply:
Didn't put anything actionable here :P
To make it clear, I'm saying that the current situation is ok for the PR, but we should untangle it soon.